modbus_exporter.go

```diff
@@ -31,10 +33,29 @@ import (
 	"github.com/lupoDharkael/modbus_exporter/modbus"
 )
 
+// ModbusRequestStatusType possible status of the modbus request
+type ModbusRequestStatusType string
+
+const (
+	// ModbusRequestStatusOK successful
+	ModbusRequestStatusOK ModbusRequestStatusType = "OK"
+	// ModbusRequestStatusErrorSock error opening socket connection
+	ModbusRequestStatusErrorSock ModbusRequestStatusType = "ERROR_SOCKET"
+	// ModbusRequestStatusErrorTimeout connection established but no response from modbus device
+	ModbusRequestStatusErrorTimeout ModbusRequestStatusType = "ERROR_TIMEOUT"
+	// ModbusRequestStatusErrorParsingValue error parsing value received
+	ModbusRequestStatusErrorParsingValue ModbusRequestStatusType = "ERROR_PARSING_VALUE"
+)
+
+var (
```

Comment:
I would try to avoid global metric variables.

Reply:
Agreed.

Reply:
Any update here?

Reply:
@daviddetorres